### DISCUSSION TITLE
Align folder arrow with file icon

### DISCUSSION THREAD

**Discussion by piedrahitapablo** [nvim-tree/nvim-tree.lua#2016] [Q&A] [answered]
Hi, I'm trying to align the folder arrow with the file icon, but I can't find a way to do it so I'm wondering if this is possible at all. 

This VS Code screenshot shows what I want to achieve:

![Screen Shot 2023-02-23 at 11 09 31 AM](https://user-images.githubusercontent.com/6783539/220964278-3556f501-ed90-47d6-bb7c-d5d788bc8195.png)

And this is what I have using nvim-tree:

![Screen Shot 2023-02-23 at 11 13 21 AM](https://user-images.githubusercontent.com/6783539/220965234-2d28bfd9-2296-4a6a-9ca6-047869bcf449.png)

And these are the settings that I'm using:

```
tree.setup {
    disable_netrw = true,
    hijack_netrw = true,
    hijack_cursor = true,
    view = {
        side = 'left',
        number = true,
        relativenumber = true,
        width = 30,
        hide_root_folder = true,
    },
    renderer = {
        indent_width = 1,
        indent_markers = {
            enable = true,
            inline_arrows = true,
            icons = {
                corner = "",
                edge = "│",
                item = "",
                bottom = "─",
                none = " ",
            },
        },
        icons = {
            git_placement = "after",
            show = {
                folder = false,
            },
        },
    },
    git = {
        ignore = false,
    },
    filters = {
        dotfiles = false,
    },
    diagnostics = {
        enable = true,
        show_on_dirs = true,
    },
    modified = {
        enable = true,
    }
}
```

Thanks!

**Comment by gegoune** [comment]
This is all down to your terminal and how it displays glyphs as well as your font and glyphs it has. Unfortunately there is nothing that nvim-tree can do here.

**Reply by piedrahitapablo** [reply]
I'm not sure because it also happens on the screenshot on the README, for example, the files marked with red are children of `actions` and the file icons are not aligned with the folder arrow of their siblings:

![jhksgkjdhvfdkhjs](https://user-images.githubusercontent.com/6783539/220976007-95c0f646-f9ea-4e23-b6aa-9fbd9a7c4c49.png)

what I want to do is this:

![195207026-f3434ba1-dc86-4c48-8ab3-b2efc3b85227](https://user-images.githubusercontent.com/6783539/220976061-4aa100b6-cfd7-4e71-9d7e-27522f072c96.png)

now the file icons are aligned with the folder arrows of their siblings

**Reply by gegoune** [reply]
Ah, apologies, I misunderstood what you were asking for, I assumed horizontal alignment of icons themselves.

Not sure if what you are asking for is currently achievable, would have to go through various options, but it definitely should be made possible if currently isn't.

**Reply by piedrahitapablo** [reply]
no, no problem, I didn't explain very well. Thanks for your answers, I'll wait to see if someone knows if it's possible right now or easily added to the codebase without breaking other things.

I'm just learning neovim and lua, but happy to make a PR if someone points me in the right direction.

**Reply by alex-courtis** [reply]
Try this:

```lua
  renderer = {
    indent_markers = {
      enable = true,
    },
    icons = {
      show = {
        folder_arrow = false,
      },
      glyphs = {
        folder = {
		  default = "",
		  open = "",
        },
      },
    },
  },
```

**Reply by piedrahitapablo** [reply]
Thanks @alex-courtis, it worked like a charm! I created a new comment with the answer and tagged you, let me know if you want to create the comment yourself and I can delete it

**Reply by alex-courtis** [reply]
I'd be grateful if you added this Tip to the [wiki](https://github.com/nvim-tree/nvim-tree.lua/wiki)

**Comment by piedrahitapablo** [comment, marked as answer]
Answer by @alex-courtis:

> Try this:
> 
> ```lua
>   renderer = {
>     indent_markers = {
>       enable = true,
>     },
>     icons = {
>       show = {
>         folder_arrow = false,
>       },
>       glyphs = {
>         folder = {
> 		  default = "",
> 		  open = "",
>         },
>       },
>     },
>   },
> ```
